Branch-SDK/src/main/java/io/branch/referral/ServerRequest.java

```diff
@@ -668,17 +663,4 @@ boolean isInitializationOrEventRequest() {
         }
         return false;
     }
-
-    protected static void addPartnerParams(JSONObject body, BranchPartnerParameters partnerParams) throws JSONException {
```

Comment:
not used anywhere, should have been cleaned up after implementing hashed Facebook param integration.